examples/framework/preferences/minimalEditor/README.md

```diff
@@ -0,0 +1,9 @@
+
+## Minimal Preferences Editor ##
+
+This folder provides an example of a minimal but fully functional preferences editor. It has:
+- only one preference
+- no enactor
+- no localization (Note that this is not best practice!)
+
+This code is the basis for the Preferences Framework tutorial.
```

Comment:
Do we know where the tutorial is going to live yet, it would be good to have link to it.